site/content/vendir/docs/develop/vendir-spec.md

```diff
@@ -23,6 +23,11 @@ directories:
   - # path lives relative to directory path # (required)
     path: github.com/cloudfoundry/cf-k8s-networking
 
+    # skip fetching if the config for this path has not changed since the last sync
+    # optional, `false` by default, available in v0.35.1+
```

Comment:
```suggestion
    # optional, `false` by default, available since v0.36.0
```

Reply:
I took version 0.35.1 from the PR page on GitHub, I'll update it to 0.36.0. But I've noticed that there are a bunch of vendir-spec.md files for different versions. Should the previous versions be updated too?

Reply:
Never mind, the latest version there is 0.35.

I have applied your suggestion, thanks!

Reply:
> But I've noticed that there are a bunch of vendir-spec.md files for different versions.

Yeah, it seems like we are lagging behind in the doc releases 😞 . Just develop should be fine for now, once we make the doc releases, it will get updated from develop.